src/cdk/drag-drop/drag-ref.ts

```diff
@@ -255,20 +241,12 @@ export class DragRef<T = any> {
    * Emits as the user is dragging the item. Use with caution,
    * because this event will fire for every pixel that the user has dragged.
    */
-  moved: Observable<{
+  moved = new Subject<{
```

Comment:
This is a breaking API change, because the type used to be `Observable`. It can be avoided by keeping the internal observable and assigning `moved` to `_moveEvents.asObservable()`.

Reply:
Ok, I'll update PR